app.js

```diff
@@ -75,8 +71,50 @@ CookieStand.prototype.cookieStandTableCells = function() {
   cookieTable.appendChild(trEl);
 };
 
-pike.cookieStandTableCells();
-seaTac.cookieStandTableCells();
-seaCent.cookieStandTableCells();
-capHill.cookieStandTableCells();
-alki.cookieStandTableCells();
+// render the header and table cells with conttent
+var render = function() {
+  makeHeaderCells();
+  for(var i = 0; i < stands.length; i++) {
+    stands[i].cookieStandTableCells();
+  }
+};
+
+render();
+// Grab form in html
+var cookieForm = document.getElementById('cookie_form');
+// var cookieFormArray = [];
+
+// event handler
+function handleCookieForm(event) {
+  event.preventDefault();
+// console.log('I heard a click');
+
+// pulling data from form into handler
+  var locationName = document.getElementById('location_name');
+  var minCust = document.getElementById('min_cust');
+  var maxCust = document.getElementById('max_cust');
+  var avgCookies = document.getElementById('avg_cookies');
+  var address = document.getElementById('add_address');
+  var phoneNum = document.getElementById('phone_num');
+
+// creating new instance of a cookie stand
+  var newCookieStand = new CookieStand(locationName.value, minCust.value, maxCust.value, avgCookies.value, address.value, phoneNum.value);
+
+  newCookieStand.cookieStandTableCells();
+
+  console.log(stands);
```

Comment:
When pushing code to production, you want to kill `console.log` lines like this.